src/logic.js

```diff
@@ -487,8 +489,26 @@ define([
                     load[path] = [prop];
                 }
             });
-
-            return {load: load};
+            var saveToCaseQuestions = _.filter(this.form.getMugList(), function(mug) {
+                return mug.__className === "SaveToCase";
```

Comment:
> To simplify the interface, it might make sense to add a function (something like `mug.options.getCaseSaveData(mug)`) to the mug options object that returns `{case_type: ..., properties: [...], create: ..., close: ...}` so the implementation below, which looks specific to the SaveToCase mug type, can be moved into saveToCase.js. If you go that route, the presence of that function could be the flag used here.

I really like this idea - will give it a shot.